src/entity/Feature.ts

```diff
@@ -31,4 +31,10 @@ export class Feature {
 
   @Column({ type: 'smallint', default: FeatureValue.Allow })
   value: FeatureValue = FeatureValue.Allow;
+
+  @Column({ length: 36, nullable: true })
+  invitedById: string;
+
+  @ManyToOne(() => User, { lazy: true, nullable: true })
```

Comment:
We need to add a setting here that in case the user gets deleted this will be set to null. I don't remember the exact property but there's something. Remember to regenerate the migration

Reply:
Good point!
Check `deleteUser` in the user directive please.
There are some examples of author being set to null for instance.

Reply:
I think it should be CASCADE: null. Can't exactly remember the right property though.